packages/xstate-inspect/src/types.ts

```diff
@@ -6,13 +6,17 @@ export type MaybeLazy<T> = T | (() => T);
 
 export type ServiceListener = (service: Interpreter<any>) => void;
 
+export type Replacer = (key: string, value: any) => any;
+
 export interface InspectorOptions {
-  url: string;
-  iframe: MaybeLazy<HTMLIFrameElement | null | false>;
-  devTools: MaybeLazy<XStateDevInterface>;
+  url?: string;
+  iframe?: MaybeLazy<HTMLIFrameElement | null | false>;
+  devTools?: MaybeLazy<XStateDevInterface>;
+  serialize?: Replacer | undefined;
```

Comment:
nit: the different naming of the property and the type is slightly confusing

Reply:
It is a replacer (copying the same type from `JSON.stringify(value, replacer, spacing)`), and `serialize` is how it's commonly known.